ports/mimxrt/boards/MIMXRT1052.ld

```diff
@@ -1,14 +1,25 @@
 /* Memory configuration */
+#if BOARD_FLASH_RESERVED
+reserved_size = BOARD_FLASH_RESERVED;
+#endif
+
+#if BOARD_FLASH_TYPE==qspi_nor
+flash_start         = 0x60000000;
+#elif BOARD_FLASH_TYPE==hyperflash
 flash_start         = 0x60000000;
+#else
+#error Unknown BOARD_FLASH_TYPE
+#endif
+flash_size 			= BOARD_FLASH_SIZE;
```

Comment:
looks like there are tabs in this line, please replace them with spaces

Reply:
fixed.